.gitignore

```diff
@@ -136,4 +136,13 @@ dmypy.json
 .pyre/
 
 
-.vscode
\ No newline at end of file
+.vscode
+
+# Emacs
+*~
+\#*\#
+
+# Vector DBs
```

Comment:
These are specific to particular vector-dbs, and we wouldn't want to have an ever-expanding list of patterns here. It's protecting against adding everything to git-tracking, which is anyway a bad practice. I would remove these, in favor of requiring people follow best practice of not adding everything to git tracking.

Reply:
I'll take care of this in my mods to this PR